pkg/hubble/relay/pool/option.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pool
+
+import (
+	"time"
+
+	"github.com/cilium/cilium/pkg/backoff"
+	"github.com/cilium/cilium/pkg/defaults"
+	peerTypes "github.com/cilium/cilium/pkg/hubble/peer/types"
+	"github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+	"github.com/sirupsen/logrus"
+
+	"google.golang.org/grpc"
+)
+
+// DefaultOptions is the reference point for default values.
+var DefaultOptions = Options{
+	PeerServiceAddress: "unix://" + defaults.HubbleSockPath,
+	PeerClientBuilder: peerTypes.LocalClientBuilder{
+		DialTimeout: 5 * time.Second,
+	},
+	ClientConnBuilder: GRPCClientConnBuilder{
+		DialTimeout: 5 * time.Second,
+		Options: []grpc.DialOption{
+			grpc.WithInsecure(),
```

Comment:
I assume this also requires a `grpc.WithInsecure(), // FIXME: remove once mTLS is implemented`?

Reply:
I think this case is fine assuming the connection is local on a Unix Domain Socket (as is the default).